test/spec/functional/chaiPromises.js

```diff
@@ -2,16 +2,14 @@ describe('chai-as-promised', function() {
 
     /**
      *  Remove the `should` global on Object.prototype to allow chai.should for these tests,
-     *  and set up chai, chai-as-promised and chai.should properly
+     *  and set up chai and chai.should properly
      */
 
     require('should').noConflict();
 
     var chai = require('chai');
-    var chaiAsPromised = require('chai-as-promised');
 
     chai.Should();
-    chai.use(chaiAsPromised);
```

Comment:
Oh, this is why I took it out from here, because I added it elsewhere.
(Sorry; I'm in a restaurant doing this on my phone.)
On Sep 11, 2015 18:48, "Christian Bromann" <notifications@github.com> wrote:

> In test/spec/functional/chaiPromises.js
> <https://github.com/webdriverio/webdriverio/pull/779#discussion_r39326111>
> :
>
> >
> >      chai.Should();
> > -    chai.use(chaiAsPromised);
>
> I see. Well we have that chaiPromise test to make sure that
> transferPromiseness works. I guess if we just add the chai stuff in there
> again we still proper test this function.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webdriverio/webdriverio/pull/779/files#r39326111>.
>